.idea/src/lesson15/HomeTask/ZooClub.java

```diff
@@ -0,0 +1,44 @@
+package lesson15.HomeTask;
+
+import java.util.*;
+
+
+public class ZooClub   {
+
+    public static void main(String[] args) {
+
+        List<Person> personList = new LinkedList<>();
+        personList.add(new Person("Petya","25"));
+        personList.add(new Person("Oksana", "20"));
+        personList.add(new Person("Sonya", "23"));
+
+        List<Animal> animalList = new ArrayList<>();
+        animalList.add(new Animal("Cat", "Sebastian"));
+        animalList.add(new Animal("Dog", "Godzilla"));
+        animalList.add(new Animal("Chinchilla", "Mika"));
+
+        Map<String, String> map = new HashMap<>();
```

Comment:
you need to have  Map<Person, List<Pet>>, not just Map<String, String>, a person can have many animals